packages/react-components/src/components/table-output-component.tsx

```diff
@@ -674,7 +708,8 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                     // non-contiguous row found, stop searching in cache
                     currRowIndexFound = false;
                 }
-                if (currRowIndexFound && !isFound && rowNode.rowIndex && rowNode.data && rowNode.data['isMatched']) {
+                // only checking 'rowNode.rowIndex' below makes its '=== 0' case false:
+                if (currRowIndexFound && !isFound && (rowNode.rowIndex || rowNode.rowIndex === 0) && rowNode.data && rowNode.data['isMatched']) {
```

Comment:
Still, `rowNode.data.isMatched` is giving false positives due to the issue with `searchEvents`.

Reply:
I understand, however this fix remains necessary within its smaller scope, I think.